OvieMudi/Questioner__
# #163073978 Admin can access a page for adding new meetups

#### What does this PR do?
- Adds a dialog box with inputs for admin to add new meetups
- Change the color theme of the sign in, signup, and dashboard pages
- Fix issues of fonts not displaying correctly
- Logically structure the folder tree for UI templates and API routes

#### Description of Task to be completed?
- Ensure that the page at /UI/adm_dashboard has a visible button/icons to add a new meetup
- Ensure that when the button/icon is clicked, a dialog box with text inputs is displayed for adding new meetups
- Ensure all API router files are in the "routes" folder
- Ensure all stylesheets and script files are their respective folders

#### How should this be manually tested?
- Open page link URL at /UI/adm_dashboard and ensure all icons
- Ensure Admin can access an input for adding new meetups
#### Any background context you want to provide?
- This is to solve issues raised by the Bootcamp LFA on frontend

#### What are the relevant pivotal tracker stories?
[#163073978](https://www.pivotaltracker.com/story/show/163073978)

#### Screenshots
![admin add meetup desktopthis](https://user-images.githubusercontent.com/42499892/50956011-e5991c00-14ba-11e9-8dae-ad55535b2081.PNG)

![admin add meetup responsive](https://user-images.githubusercontent.com/42499892/50955961-c26e6c80-14ba-11e9-83d0-31f48ec9abb8.PNG)
